samples/apps/tinyRA/tinyra/run_tab.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+from autogen import config_list_from_json
+from autogen import AssistantAgent, UserProxyAgent
+import tui
+
+msgid = None
+# get the first arg
+if len(sys.argv) > 1:
+    msgid = int(sys.argv[1])
+    if msgid < 1:
+        msgid = None
+
+if msgid is None:
+    print("Please specify a valid message id")
+    exit(1)
+
+config_list = config_list_from_json("OAI_CONFIG_LIST")
+
+assistant = AssistantAgent("tinyra", llm_config={"config_list": config_list})
```

Comment:
One suggestion (feel free to reject) is to use CompressibleAgent here, because it may be easily out of tokens if the user uses it for a long time.

Reply:
I added a todo for this in the py file.